src/Table/index.ts

```diff
@@ -0,0 +1,6 @@
+
+(window as any).global = window;
+// @ts-ignore
+window.Buffer = window.Buffer || require("buffer").Buffer;
```

Comment:
why? can you put a comment describing why we need this

Reply:
Oh, the recent node release has removed the array buffer polyfill i believe. It has been moved to a separate dependency  [here](https://www.npmjs.com/package/buffer) and you need to install it if you need it. So the code here simply check if the browser has it, otherwise require it.